aspnet/mvc/models/validation.rst

```diff
@@ -1,4 +1,4 @@
-Model Validation 
+Model Validation
```

Comment:
Will probably help to add `?w=1` to the URL. That removes changed whitespace from the equation.